src/main.js

```diff
@@ -7,6 +7,8 @@ Vue.config.productionTip = false;
 
 import * as VueGoogleMaps from 'vue2-google-maps';
 
+export const eventBus = new Vue();
```

Comment:
It’s really interesting to me that the eventBus is a whole new instance of Vue. I would think this would require a lot of resources and slow things down, but it seems like the app is still pretty lightweight!